src/status_im2/common/standard_authentication/standard_auth/slide_button/view.cljs

```diff
@@ -4,17 +4,18 @@
     [quo.theme :as quo.theme]
     [react-native.core :as rn]
     [reagent.core :as reagent]
-    [status-im2.common.standard-authentication.standard-auth.authorize :as authorize]))
+    [status-im2.common.standard-authentication.standard-auth.authorize :as authorize]
+    [utils.re-frame :as rf]))
 
 (defn- view-internal
   [_]
-  (let [reset-slider? (reagent/atom false)
-        on-close      #(reset! reset-slider? true)]
-    (fn [{:keys [biometric-auth?
-                 track-text
+  (let [reset-slider?   (reagent/atom false)
+        on-close        #(reset! reset-slider? true)
+        auth-method     (rf/sub [:auth-method])
+        biometric-auth? (= auth-method "biometric")]
```

Comment:
Maybe the auth-method string should be a keyword, or even a namespaced keyword like `:auth-method/biometric`, `:auth-method/password`